CONTRIBUTING.rst

```diff
@@ -152,220 +133,208 @@ Debian/Ubuntu
 
    -  Declare an environment variable ``AWS_LIBS`` for this directory
 
-   .. code:: sh
+      .. code:: sh   
 
-      export AWS_LIBS=/tmp/lib/aws
+         export AWS_LIBS=/tmp/lib/aws
 
-   -  Percona Server for MongoDB is built with AWS SDK CPP 1.8.187
+   -  Percona Server for MongoDB is built with AWS SDK CPP 1.9.379
       version. Switch to this version
 
-   .. code:: sh
+      .. code:: sh   
 
-      cd aws-sdk-cpp && git checkout 1.8.187
+         cd aws-sdk-cpp && git checkout 1.9.379
 
    -  It is recommended to keep build files outside the SDK directory.
       Create a build directory and navigate to it
 
-   .. code:: sh
+      .. code:: sh   
 
-      mkdir build && cd build
+         mkdir build && cd build
 
    -  Generate build files using ``cmake``
 
-   .. code:: sh
+      .. code:: sh   
 
-      cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ONLY="s3;transfer" -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
+         cmake .. -DCMAKE_BUILD_TYPE=Release '-DBUILD_ONLY=s3;transfer' -DBUILD_SHARED_LIBS=OFF -DMINIMIZE_SIZE=ON -DCMAKE_INSTALL_PREFIX="${AWS_LIBS}"
 
    -  Install the SDK
 
-   .. code:: sh
+      .. code:: sh   
 
-      make install
+         make install
 
 6. Build Percona Server for MongoDB
 
    -  Change directory to ``percona-server-mongodb``
 
-   .. code:: sh
+      .. code:: sh
 
-      cd percona-server-mongodb
+         cd percona-server-mongodb
 
    -  Build Percona Server for MongoDB from ``buildscripts/scons.py``.
+      
+      - With features included in Percona Supported Builds:
+      
+      .. code:: sh   
 
-   .. code:: sh
+         buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup --enable-fipsmode CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
+
+      - Without features included in Percona Supported Builds:
 
-      buildscripts/scons.py -j$(nproc --all) --jlink=2 --disable-warnings-as-errors --ssl --opt=on --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib" install-mongod
+      .. code:: sh   
 
-This command builds only the database. Other available targets for the
-``scons`` command are: - ``mongod`` - ``mongos`` - ``mongo`` - ``core``
-(includes ``mongod``, ``mongos``, ``mongo``) - ``all``
+         buildscripts/scons.py --disable-warnings-as-errors --release --ssl --opt=on -j$(nproc --all) --use-sasl-client --wiredtiger --audit --inmemory --hotbackup CPPPATH="${AWS_LIBS}/include" LIBPATH="${AWS_LIBS}/lib ${AWS_LIBS}/lib64" install-mongod install-mongos install-mongo
+
+This command builds the core of the database. Other available targets for the
+``scons`` command are: 
+
+- ``mongod`` 
+- ``mongos`` 
+- ``mongo`` 
+- ``core`` (includes ``mongod``, ``mongos``, ``mongo``) 
+- ``all``
```

Comment:
Should be install-mongod, install-mongos, install-mongo, install-core, install-all